src/components/SMLinks/index.jsx

```diff
@@ -3,7 +3,8 @@ export function SMLinks({ links }) {
   return (
     <SMContainer>
       {links.map(({ id, path, Icon }) => (
-        <StyledSMLinks href={path} key={id}>
+        <StyledSMLinks href={`https://${path}`} key={id}>
+          `
```

Comment:
What is going on with this backtic ` sign?

Reply:
done